packages/wonder-blocks-core/util/get-element-intersection.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+import enumerateScrollAncestors from "./enumerate-scroll-ancestors.js";
+
+/**
+ * Indicates the intersection state of an item on a single axis.
+ */
+export type AxisIntersection =
+    /**
+     * The item is partly or fully within the bounds.
+     */
+    | "within"
```

Comment:
nit: these could be line comments.

Reply:
Also, you may want to include that `"within"` implies that the intersect rect is strictly within the bounds rect.

Reply:
Not sure I understand your second comment there, @kevinbarabash. Can you elaborate? The comment that I currently have on the `within` value states that the item can be partly or fully within the bounds, which is currently correct as to the function implemented.

Reply:
d'oh... my misunderstanding